packages/common/src/Environments.ts

```diff
@@ -27,13 +27,17 @@ const defaultPenalizerConfiguration: PenalizerConfiguration = {
 }
 
 const defaultRelayHubConfiguration: RelayHubConfiguration = {
-  gasOverhead: 54771,
-  postOverhead: 19073,
+  gasOverhead: 61970,
+  postOverhead: 19037,
   gasReserve: 100000,
   maxWorkerCount: 10,
   minimumStake: 1e18.toString(),
   minimumUnstakeDelay: 1000,
-  maximumRecipientDeposit: 2e18.toString()
+  maximumRecipientDeposit: 2e18.toString(),
+  devInfo: {
```

Comment:
Why does it have to be an object? We tend to prefer flat configuration types.

Reply:
It doesn't have to be, it just seems reasonable to pack them to a struct...

Reply:
Well, I'd prefer to keep it flat.